apps/README.md

```diff
@@ -65,3 +66,13 @@ CONFIGURED_APPS += examples/hello
 endif
 ```
 
+## Optimized Buffering for SmartFs READ
+
+Using just any buffer size for reading the contents of a SmartFs file, back-to-back,
```

Comment:
@amogh-samsung Could you please make another commit within this PR, which can test this new API. This test code might be seen as an example.

Reply:
Added